test/Libraries/Microsoft.Extensions.AI.Integration.Tests/ChatClientIntegrationTests.cs

```diff
@@ -1157,7 +1157,7 @@ public virtual async Task SummarizingChatReducer_PreservesConversationContext()
         // The summarizer should have reduced the conversation
         Assert.Equal(1, chatClient.SummarizerCallCount);
         Assert.NotNull(chatClient.LastSummarizedConversation);
-        Assert.Equal(3, chatClient.LastSummarizedConversation.Count);
+        Assert.Equal(4, chatClient.LastSummarizedConversation.Count);
         Assert.Collection(chatClient.LastSummarizedConversation,
```

Comment:
@MackinnonBuck, does this collection assert not also need to change?

Reply:
Verified locally, so we should be good.